Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM alpine:3.6
```

Comment:
Do you think it might make sense to add some sort of caching tool, eg. Varnish? I don't have too much experience with hosting web services in Docker though, I don't know whether this'd be the right place.

Reply:
opcache is a good idea, but that'll depend on the setup with the themes. I think in most Docker setups you'd have the caching on a different layer. Tbh it's pretty quick as it is, some basic view caching would improve performance quite a bit and that wouldn't take much effort.